scripts/restore-storage-objects.py

```
@@ -0,0 +1,303 @@
+
+
+def request(url: str, key: str, method: str = "GET", body: bytes | None = None,
+            mime: str | None = None) -> tuple[int, bytes]:
+    req = urllib.request.Request(url, data=body, method=method)
+    req.add_header("Authorization", "Bearer " + key)
+    if mime:
+        req.add_header("Content-Type", mime)
+    if method == "POST":
+        req.add_header("x-upsert", "true")
+    try:
+        with urllib.request.urlopen(req, timeout=60) as resp:
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)